composer.json

```diff
@@ -9,19 +9,28 @@
     "license": "MIT",
     "require": {
         "php": "^7.2.5",
+        "fennb/phirehose": "dev-master",
         "fideloper/proxy": "^4.2",
         "fruitcake/laravel-cors": "^2.0",
+        "fzaninotto/faker": "^1.9.1",
         "guzzlehttp/guzzle": "^6.3",
         "laravel/framework": "^7.0",
-        "fzaninotto/faker": "^1.9.1",
         "laravel/tinker": "^2.0",
         "laravel/ui": "^2.1",
         "pusher/pusher-php-server": "~4.0",
-        "squizlabs/php_codesniffer": "*"
+        "squizlabs/php_codesniffer": "*",
+        "thujohn/twitter": "^2.2"
     },
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/onifs10/phirehose.git"
+        }
+    ],
     "require-dev": {
         "facade/ignition": "^2.0",
         "laravel/telescope": "^3.5",
+        "fzaninotto/faker": "^1.9.1",
```

Comment:
Remove faker from here, it should be in `require` only since we will be using it in the production environment.

Reply:
aiit